src/blog/components/informationMessage/InformationMessage.tsx

```diff
@@ -0,0 +1,67 @@
+import Button from "src/components/buttons/Button";
+import LinkButton from "src/components/linkButton/LinkButton";
+import Text from "src/components/text/Text";
+import { ILink, LinkType } from "studio/lib/payloads/navigation";
+import styles from "./informationMessage.module.css";
+import React from "react";
+
+export const homeLink = {
```

Comment:
how about we but our links in a util? I think we're using these in error message as well?